### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 -Emily Baquerizo and Kimberly Beyeler

### DISCUSSION THREAD

**Discussion by AEBaq** [heig-vd-dai-course/heig-vd-dai-course#346] [Show and tell]
rechercher une image selon un critère : date, nom, format, couleur(RGB, NB)

**Comment by ludelafo** [comment]
Nous comprenons l'idée générale mais pourriez-vous nous donner plus de détails ? Lorsque vous dites "rechercher une image", qu'est-ce que cela signifie ?

Peut-être des exemples préliminaires d'exécution de votre CLI nous aiderait à comprendre.

Merci !

**Reply by AEBaq** [reply]
Nous pensions trouver le chemin d'accès d'une image se trouvant dans un dossier ou dans ses potentiels sous-dossiers.

**Comment by Kurosakiim** [comment]
Nous souhaitons changer d'idée. La nouvelle idée serait d'appliquer différents filtre à une image comme passer l'image en noir et blanc, la flouter, ne faire apparaître qu'une seule couleur,...

**Reply by hadrylouis** [reply]
C'est en ordre pour nous. Merci !

**Reply by AEBaq** [reply]
Voici le lien de notre repository: https://github.com/HEIG-VD-BA5/DAI/tree/main
Et voici le hash de notre commit final : 5fd18fcf3a453d09a81ac8b25dda981a1e547c58

**Reply by ludelafo** [reply]
Bonjour @AEBaq, @Kurosakiim,

Nous n'avons pas les accès à votre repo. Merci d'ajouter @hadrylouis, @GeraudSilvestri et moi-même pour que nous puissions y avoir accès pour le corriger.

**Comment by AEBaq** [comment]
Bonjour,
Navrée de l'oubli, nous vous avons rajouté.

Meilleures salutations

Emily Baquerizo
________________________________

**Reply by hadrylouis** [reply]
Bonjour @AEBaq, @Kurosakiim,

Merci pour cela. Malheureusement, nous n'avons pas retrouvé le commit avec le bon hash sur votre repo. Avez-vous écrasé l'historique git avec un `git push --force` ?

**Reply by Kurosakiim** [reply]
Bonjour,

Il ne me semble pas que nous ayons fait un `git push --force` mais voilà notre dernier commit e49309c89ac4232ecb3f95e953127b212f68a423

Désolée de l'inconvénient

**Reply by ludelafo** [reply]
Bonsoir @AEBaq, @Kurosakiim,

Juste pour être au clair : vous nous avez mis à jour la discussion GitHub le 20 octobre à 23h43 avec un commit portant le hash `5fd18fcf3a453d09a81ac8b25dda981a1e547c58` (cf discussion suivante : https://github.com/orgs/heig-vd-dai-course/discussions/346#discussioncomment-10998789), respectant l'heure de rendu du projet.

@hadrylouis et moi-même n'avons pas été capables de retrouver ce commit dans le repository dans lequel vous nous avez invité. Vous nous indiquez maintenant que le dernier commit est `e49309c89ac4232ecb3f95e953127b212f68a423`, datant de ce matin à 7h41. De plus, le repo ne contient que 6 commits, tous datant d'hier.

Est-ce celui-ci que vous souhaitez que l'on évalue ? Si c'est le cas, soyez conscientes qu'il s'agit du coup d'un retard de 8 jours, avec -1 par jour de retard.

Je repose donc la question : où est donc le commit avec le hash `5fd18fcf3a453d09a81ac8b25dda981a1e547c58` ? Avez-vous une explication sur ce qu'il s'est passé ?

**Reply by AEBaq** [reply]
Bonsoir,

Le commit effectué ce matin était pour modifier une slide de la présentation. Je ne comprends pas pourquoi nous n'avons plus que 6 commits de visible... Je n'ai pas de souvenir d'avoir utilisé de ```git push --force``` également.

En retournant sur notre dernier pull request ([#40](https://github.com/HEIG-VD-BA5/DAI/pull/40/commits) de notre repository : ```negative filter created and working```), j'ai retrouvé un commit correspondant à notre dernier pull request et effectué avant la date de rendu.

Si vous l'acceptez, nous pourrions vous proposez le commit ```71d9e1d70f12e09501dc4a969eddeee1708e795d``` étant celui qui se rapproche le plus de notre version finale.

Nous sommes vraiment navrées de ce problème, je suis tout autant perdue que vous sur ce qui c'est passé avec l'historique de git...
